src/components/Sidebar.module.css

```diff
@@ -49,6 +49,9 @@
   flex-grow: 1;
   position: relative;
   overflow-y: auto;
+  & ul li h4 {
```

Comment:
This doesn't seem to target the sidebar module's title

Reply:
Working on this.

Reply:
Looks good to me. it has been applied to the targeted element

![image](https://github.com/user-attachments/assets/765b8346-c07f-40c0-aae3-cadcd082c6a5)